src/components/navigation/header/index.stories.tsx

```diff
@@ -13,14 +13,40 @@ import {
 
 import Navigation from './index';
 
+const Wrapper: typeof Navigation = ({ user, selectTenant, ...props }) => {
```

Comment:
Created a wrapper to show a small delay. Changing tenant in the apps will require a network call to refresh the token and a delay to set the cookie